src/olympia/discovery/data.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, *args, **kwargs):
                   '{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Remove blaring ads and make the Web run smoother! Adblock '
-              u'Plus can also mitigate tracking mechanisms and malware.'),
+            _(u'Remove blaring ads and make the Web run smoother with ')
```

Comment:
It won't, but that means the second part of the string is not part of the gettext call, which is bad.